### PR TITLE
add a quote

### PR DESCRIPTION
**Pull request by camenlei**
added a quote

### CODE REVIEW

**Comment by camenlei** [comment]
could you add a quote? 

Question: how do they know what order to add?

File: _posts/0000-01-02-camenlei.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (ilike-1mwtt) [inline _posts/0000-01-02-camenlei.md]: Please change filename to `0000-01-04-camenlei.md`

**Comment by camenlei** [comment]
changed file name, thank you!

**Comment by camenlei** [comment]
oops, keep pressing wrong button...

**Comment by ilike-1mwtt** [comment]
Thank you @camenlei for the contribution!

**Pull request merged**